weave/Scenes/Main.tscn

```diff
@@ -32,3 +33,13 @@ visible = false
 
 [node name="GameOverOverlay" parent="." instance=ExtResource("4_lq1d2")]
 visible = false
+
+[node name="CountdownLabel" type="RichTextLabel" parent="."]
```

Comment:
Seems like the countdown is placed manually? Wont work scaled, but so wont the game logic either so :shrug: